package.json

```diff
@@ -9,7 +9,7 @@
     "format": "yarn run format:eslint; yarn run format:prettier",
     "format:eslint": "eslint --fix --ext .js,.vue --ignore-path .gitignore .",
     "format:prettier": "prettier --ignore-path .gitignore --write **/**/*.{scss,js,yml,vue}",
-    "git-pre-push": "yarn run test",
+    "git-pre-push": "yarn run test && echo",
```

Comment:
This is going to complain when you try to merge because I stupidly committed the removal of the test.

Reply:
I think if you just delete the line from your package json and commit that, you should be able to squash and not get an error.